### PR TITLE
Fix shutdown items flushing

### PR DESCRIPTION
**Pull request by coryvirok**

### CODE REVIEW

**Comment by jondeandres** [comment]
@coryvirok I've force pushed to this, hope you didn't push anything :-|.

I think I've fixed this.

code to test

```javascript
var rollbar = require('./rollbar');

rollbar.init('8802be7c990a4922beadaaefb6e0327b', {
  handler: 'nextTick'
});


var interval;
var running = true;

function doProcess() {
  if (running) {
    console.log('running!');
  } else {
    clearInterval(interval);
  }
}

interval = setInterval(doProcess, 1000);

rollbar.handleError(new Error('Illustrate crasher 1'));
rollbar.handleError(new Error('Illustrate crasher 2'));
rollbar.handleError(new Error('Illustrate crasher 3'));
rollbar.handleError(new Error('Illustrate crasher 4'));

function exit() {
  process.exit(0);
}

rollbar.shutdown(function() {
  running = false;
  console.error('shutdown, everything should be flushed/posted');
  setTimeout(exit, 0)
});
```

File: package.json [hunk 1]
```diff
@@ -24,11 +24,13 @@
...
     "node-uuid": "1.4.x",
     "lru-cache": "~2.2.1",
-    "json-stringify-safe": "~5.0.0"
+    "json-stringify-safe": "~5.0.0",
+    "async": "~0.9.0"
   },
   "devDependencies": {
     "express": "*",
     "jade": "~0.27.7",
+    "jslint": "^0.9.0-pre006",
```
Comment (jondeandres) [inline package.json]: @coryvirok seems that the problem for the tests not passing is this.

Can we remove it? Or is needed?

Comment (jondeandres) [inline package.json]: I've removed it and tests passing here https://travis-ci.org/rollbar/node_rollbar/builds/57837278.

**Comment by coryvirok** [comment]
Alright, I think this is ready to go. @jondeandres if these changes look good, I'll merge to master and bump the minor version.

**Comment by jondeandres** [comment]
@coryvirok awesome :+1:

**Comment by coryvirok** [comment]
Thanks @twalling. This change fixes - https://github.com/rollbar/node_rollbar/commit/01d512a2b73b38ef1e31dc2485f482f870f55839

**Pull request merged**